src/betterproto/plugin/main.py

```diff
@@ -13,6 +13,7 @@ def main():
     data = sys.stdin.buffer.read()
 
     # Parse request
+    # import extension_pb2
```

Comment:
If you uncomment this, you get:

```console
Generated output for 'custom_options'
[extension] {
  max_size: 40
}
```

When running `python -m tests.generate -v`. Otherwise you don't (because protoc does not pick up the custom options)